.env

```diff
@@ -1,3 +1,4 @@
-REACT_APP_CHAIN_ID="1"
+REACT_APP_CHAIN_ID="3"
```

Comment:
this should still be `1` - override the value in `.env.local` for local testing rather than here

Reply:
There is invalid blocktag error after merging the master. It doesn't work on Ropsten now. I will change it back to mainnet once testing on Ropsten.